src/proto/CMakeLists.txt

```
@@ -7,6 +7,10 @@ if (${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
     list(APPEND CMAKE_PREFIX_PATH "${Protobuf_PREFIX_PATH}")
 endif()
 find_package(Protobuf REQUIRED)
+if (NOT "${Protobuf_VERSION}" VERSION_LESS 4.22.y)
```

Is `4.22.y` a specific protobuf library version?

---

> Is `4.22.y` a specific protobuf library version?

@bukepo  4.22.y, means main version is 22 and major verion "y" can be any one. I've tested the 4.22.1 and 4.22.99 (only for test), the VERSION_LESS can work with it.

---

```suggestion
if (NOT "${Protobuf_VERSION}" VERSION_LESS 4.22)
```
I think you can use `4.22` instead.

---

> I think you can use `4.22` instead.

Tested success in local, commit this with suggestion  and update the PR.